server/events_test.go

```diff
@@ -2535,6 +2535,73 @@ func TestServerEventsStatszSingleServer(t *testing.T) {
 	checkSubsPending(t, sub, 1)
 }
 
+func TestServerEventsReload(t *testing.T) {
+	conf := createConfFile(t, []byte(`
+		listen: "127.0.0.1:-1"
+		accounts: {
+			$SYS { users [{user: "admin", password: "p1d"}]}
+			test { users [{user: "foo", password: "bar"}]}
+		}
+		ping_interval: "100ms"
+	`))
+	opts := LoadConfig(conf)
+	opts.Trace = true
+	opts.Debug = true
+	opts.TraceVerbose = true
+	s := RunServer(opts)
+	defer s.Shutdown()
+	subject := fmt.Sprintf("$SYS.REQ.SERVER.%s.RELOAD", s.info.ID)
```

Comment:
Use const here vs string literal?

Reply:
Hardcoding the `"$SYS..."` formats seems to be the standard operating procedure in events_test; 1/5 tests should have these strings independently defined (hard-coded?) anyway.

Reply:
If we have a const for doing the template we should use that..